src/tests/core/fixtures/fixtures_global.c

```diff
@@ -0,0 +1,112 @@
+#include "fixtures_tx.h"
+#include "fixtures_global.h"
+#include "utxo_pool.h"
+#include "wallet_pool.h"
+#include "blockchain.h"
+#include "mempool.h"
+#include "utxo_to_tx.h"
+#include "init_db.h"
+
+#include <string.h>
+
+static ComposedFixture *composition = NULL;
+
+int fixture_setup_compose(void **state) {
+  composition = *state;
+  *state = NULL;
+
+  int i = 0;
```

Comment:
Unclear what composing means or composed fixture (Very small comments in the headers might clear this up)